src/condor_scripts/condor_watch_q

```diff
@@ -872,6 +900,12 @@ class JobStateTracker:
                     )
                     continue
 
+                if self.filter_ids is not None and event.cluster not in self.filter_ids:
+                    continue
+
+                if self.since is not None and event.cluster < self.since[0]:
```

Comment:
`self.since` seems like it really ought to be an integer.  What does this even do?

Reply:
When I made the PR, I had no clue why the argparser was return a list. That is why I had to access the first member. After some fresh mind research, it turns out this behavior is due to the inclusion of nargs=1. I have fixed this to be just a pure integer value

Reply:
Thanks for the explanation!  I had assumed that `nargs = 1` would result in a scalar, but I can see why they'd made `nargs = None` a scalar instead.